### ISSUE TITLE
Get rid of pg_exttable.fmterrtbl

### ISSUE THREAD

**Issue by ascherbaum-pivotal** [greenplum-db/gpdb#1170]
The pg_exttable.fmterrtbl column stored the OID of the error table, but without an error table it is just set to the OID of the external table. That is not necessary, there are other columns which indicate if error logging is enabled. Therefore this column can be removed.

**Comment by deepesh1081** [comment]
This column was used to determine the error table table type, If we are going to drop error tables and only keep error log, yes this is not needed and can be removed.

**Comment by ascherbaum-pivotal** [comment]
According to @hlinnaka on Slack the code for error tables is no longer in v5.

**Comment by Eulerizeit** [comment]
@yydzero This is causing issues with pg_dump

**Comment by HaozhouWang** [comment]
@Eulerizeit What's the issue? Do you mean removing this will cause issue?

**Comment by Eulerizeit** [comment]
It should be removed.

Rob

**Comment by yydzero** [comment]
Hi @ivannovick 

Before we do any change, could you confirm that error table will be removed completely from 5.0?

Do we have document about EOF this?

**Comment by ivannovick** [comment]
I believe error table is completely removed from 5.0  @Eulerizeit can you confirm?

**Comment by ashwinstar** [comment]
Error Tables were completely removed from 5.0 long time back.

```
commit 3fd4bd9c6634d90f23099bdce75a910ba2813fad
Author: Abhijit Subramanya <asubramanya@pivotal.io>
Date:   Thu Jan 14 17:33:47 2016 -0800

    Remove Error tables functionality in favor of the error log functionality.

    Users should use error log functionality for error handling in external tables
    and during copy since it is more robust and reliable than the old error table
    functionality.
```

**Comment by yydzero** [comment]
Thanks for the confirmation

**Comment by adam8157** [comment]
Will do this in next catalog window, it's frozen right now.

**Comment by hlinnaka** [comment]
@adam8157, ping? Can we please make sure this gets done in GPDB6?

**Comment by yydzero** [comment]
assign to Adam & Brian

**Comment by adam8157** [comment]
Thanks for the notification, story created, will fix it next week.

**Comment by adam8157** [comment]
Fixed by #4673
